src/Logging/Logging.Console/src/ConsoleLogger.cs

```diff
@@ -40,6 +42,18 @@ internal ConsoleLogger(string name, ConsoleLoggerProcessor loggerProcessor)
             _queueProcessor = loggerProcessor;
         }
 
+        internal ConsoleLogger(string name, ConsoleLoggerProcessor loggerProcessor, ISystemClock systemClock)
```

Comment:
Just forward the other ctor to this one:
```c#
internal ConsoleLogger(string name, ConsoleLoggerProcessor loggerProcessor) : this(name, loggerProcessor, new SystemClock()) {}
```
No need to duplicate logic.

Or as they are internal, just merge them with default param `systemClock = null` and the `_systemClock = systemClock ?? new SystemClock();`.

Reply:
Sounds good to me, sloppy change on my side - Sorry!